COPYRIGHT

```diff
@@ -0,0 +1,124 @@
+GAP, a system for computational discrete algebra.
+
+GAP is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free
+Software Foundation; either version 2 of the License, or (at your
+option) any later version.
+
+GAP is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GAP; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+Copyright (C) 1988-2018 by its authors, which include:
+
+* Ákos Seress
+* Alex Wegner
+* Alexander Hulpke
+* Alexander Konovalov
+* Alice Niemeyer
+* Andreas Distler
+* Andrew Solomon
+* Ansgar Kaup
+* Arne Storjohann
+* Bernhard Reinke
+* Bettina Eick
+* Bill Allombert
+* Björn Assmann
+* Blair Archibald
+* Bohdan Majewski
+* Burkhard Höfling
+* Charles R.B. Wright
+* Chris Jefferson
+* Chris Wensley
+* Christian Sievers
+* Christopher Russell
+* Claus Fieker
+* Colva M. Roney-Dougal
+* Craig A. Struble
+* David Joyner
+* Derek F. Holt
+* Dima Pasechnik
```

Comment:
you can remove Dima Pasechnik, and leave Dmitrii Pasechnik only there. 
Dima is a short form of Dmitrii, nothing more, and I am known under both of them.

Reply:
Oops, indeed, I overlooked it. Thanks, will be fixed